src/arb.rs

```diff
@@ -54,20 +54,15 @@ impl Arbitrary for Cid {
     fn arbitrary<G: Gen>(g: &mut G) -> Self {
         let version: Version = Arbitrary::arbitrary(g);
         let v0 = version == Version::V0;
-        let codec: Codec = if !v0 {
-            // v1 supports arbitrary encodings
-            Arbitrary::arbitrary(g)
-        } else {
-            // v0 does only support DagProtobuf encoding
-            Codec::DagProtobuf
-        };
-        let hash: Multihash = if !v0 {
-            Arbitrary::arbitrary(g)
-        } else {
+        if v0 {
```

Comment:
(nitpick, no need to change it) Due to the API changes, this could be written as `if version == Version::V0 {`